apps/io-fast-login/package.json

```diff
@@ -0,0 +1,73 @@
+  "description": "Fast Login functions",
+  "main": "./dist/main.js",
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "shx rm -rf dist && npm run generate",
+    "postbuild": "dependency-check package.json --no-dev --missing ./dist/**/*.js",
+    "build": "yarn prebuild && tsc && yarn postbuild",
+    "watch": "tsc -w",
+    "extensions:install": "func extensions install",
+    "start": "dotenv -e .env func start --javascript",
+    "predev": "npm-run-all build",
+    "dev": "npm-run-all --parallel start watch",
+    "test": "vitest run",
+    "test:coverage": "vitest run --coverage",
+    "test:watch": "vitest",
+    "lint": "yarn lint:check --fix",
+    "lint:check": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
+    "lint-api": "find ./api -name '*.yaml' -exec swagger-cli validate {} \\;",
+    "api:bundle": "bundle-api-spec -i api/internal.yaml.template -o api/internal.yaml -V $npm_package_version",
+    "pregenerate": "yarn api:bundle",
+    "generate": "yarn pregenerate && yarn generate:definitions-internal && yarn generate:models && yarn generate:client-fn-lollipop && yarn generate:client-backend-internal",
+    "generate:definitions-internal": "rimraf ./src/generated/definitions/internal && shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal ",
+    "generate:models": "rimraf ./src/generated/definitions/models && shx mkdir -p ./src/generated/definitions/models && gen-api-models --api-spec ./api/definitions.yaml --no-strict --out-dir ./src/generated/definitions/models ",
+    "generate:client-fn-lollipop": "rimraf ./src/generated/definitions/fn-lollipop && shx mkdir -p ./src/generated/definitions/fn-lollipop && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-lollipop/master/openapi/external.yaml --no-strict --out-dir ./src/generated/definitions/fn-lollipop --request-types --response-decoders --client",
+    "generate:client-backend-internal": "rimraf ./src/generated/definitions/backend-internal && shx mkdir -p ./src/generated/definitions/backend-internal && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v13.17.0/openapi/generated/api_session.yaml --no-strict --out-dir ./src/generated/definitions/backend-internal --request-types --response-decoders --client",
+    "dist:modules": "modclean -r -n default:safe && yarn install --production",
+    "predeploy": "npm-run-all build dist:*",
+    "preversion": "auto-changelog  --config .auto-changelog.json --unreleased --commit-limit false --stdout --template preview.hbs",
+    "version": "npm-run-all bump:*",
+    "bump:api": "npm-run-all api:* && git add api/*.yaml",
+    "bump:changelog": "auto-changelog -p --config .auto-changelog.json --unreleased && git add CHANGELOG.md"
+  },
+  "devDependencies": {
+    "@pagopa/eslint-config": "^3.0.0",
+    "@types/express": "^4.17.12",
+    "@types/node": "~18.13.0",
```

Comment:
this can be updated to `~20.12.2`

Reply:
Resolved in [54af889](https://github.com/pagopa/io-auth-n-identity-domain/pull/116/commits/54af8891392b63c489b7f910a76078b267f90224)